### ISSUE TITLE
not working in windows

### ISSUE THREAD

**Issue by u9520107** [koajs/static-cache#36]
using windows 7, iojs@1.5.1.
```
server.use(mount('/build', serve(__dirname)));
```
If serve = require('koa-static'), then the static files is accessible.
If serve = require('koa-static-cache'), then the static files is not accessible at all.

Here's a piece of the debug message:
```
request: /build/server.js
  koa-router GET undefined +8s
  koa-router test "/" /^(?:\/(?=$))?$/i +2ms
  koa-mount mount /jspm_packages staticCache -> false +5s
  koa-mount mount /build staticCache -> /server.js +1ms
  koa-mount enter /build/server.js -> /server.js +1ms
  koa-mount leave /build/server.js -> /server.js +2ms
```

looks like the file was found.. but somehow was not sent out, the browser request ends up in 'not found'.

**Comment by coderhaoxin** [comment]
maybe you should GET `/index.html`.

> Differences between this library and other libraries such as static:
> There is no directory or index.html support.

**Comment by u9520107** [comment]
How's that related to trying to request for /build/server.js?

**Comment by coderhaoxin** [comment]
oh, :cry: just ignored the debug msg

**Comment by coderhaoxin** [comment]
due to `path.normalize()` on window

```
path.normalize('/server.js') => '\\server.js'
```

**Comment by u9520107** [comment]
@coderhaoxin Cool. So this should be fixed in upcoming updates?

**Comment by coderhaoxin** [comment]
@u9520107 try 3.0.2

**Comment by u9520107** [comment]
@coderhaoxin Ok, confirmed fixed in windows with 3.0.2. Sorry for the delay in confirmation.
